cmd/dra-example-kubeletplugin/cdi.go

```diff
@@ -111,6 +111,8 @@ func (cdi *CDIHandler) CreateClaimSpecFile(claimUID string, devices *PreparedDev
 			spec.Devices = append(spec.Devices, cdiDevice)
 			gpuIdx++
 		}
+	default:
+		return fmt.Errorf("unknown device type")
```

Comment:
Include the actual device type in the error?

Otherwise it's just a static string and `errors.New` would be better.

Reply:
Cannot include actual device type, it's runtime-determined, there's no such field, it's either GPU or literally unknown device type:
https://github.com/kubernetes-sigs/dra-example-driver/blob/e6004d1e93a4ae502c644b69de25e917aa1416a2/api/example.com/resource/gpu/nas/v1alpha1/nas.go#L81

Reply:
Right.